src/main/kotlin/racingcar/view/InputView.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar.view
+
+object InputView {
+
+    fun readCarNames(): List<String> {
+        println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")
+        return readln().split(",")
+            .map { it.trim() }
+    }
+
+    fun readTrialCount(): Int {
+        println("시도할 횟수는 몇 회인가요?")
+
+        try {
+            return readln().toInt()
+        } catch (_: NumberFormatException) {
+            throw IllegalArgumentException("Trial count must be an integer value.")
+        }
```

Comment:
```suggestion
        return readln().toIntOrNull() ?: throw IllegalArgumentException("Trial count must be an integer value.")
        }
```
It could be more concise and Kotlin style.

Reply:
SUPER!!